android/src/main/java/com/reactnativestripesdk/PaymentOptionDisplayDataMapper.kt

```diff
@@ -0,0 +1,39 @@
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.WritableMap
+import com.stripe.android.paymentelement.EmbeddedPaymentElement
+import com.stripe.android.paymentelement.ExperimentalEmbeddedPaymentElementApi
+
+/**
+ * Serialize Stripe's PaymentOptionDisplayData into a WritableMap
+ * that can be sent over the RN bridge.
+ */
+@OptIn(ExperimentalEmbeddedPaymentElementApi::class)
+fun EmbeddedPaymentElement.PaymentOptionDisplayData.toWritableMap(): WritableMap {
+  val map = Arguments.createMap()
+
+  map.putString("label", label)
+  map.putString("paymentMethodType", paymentMethodType)
+
+  val billingMap = Arguments.createMap()
```

Comment:
Should there be a reusable place to do this logic? My guess is it already exists, and I'm scared we'd update one place and not the other.

Reply:
Surprisingly after looking around we do not have anything that turns a PaymentSheet.BillingDetails into a Map. We have something that turns a PaymentMethod.BillingDetails into a Map but nothing for PaymentSheet. I'll move this logic into a place it's easier to reuse in the future.